addons/web/static/lib/daterangepicker/daterangepicker.js

```diff
@@ -431,7 +431,10 @@
                 'click.daterangepicker': $.proxy(this.show, this),
                 // odoo cutomization, to align behavior of the daterange widget with the date picker one
                 // 'focus.daterangepicker': $.proxy(this.show, this),
-                'keyup.daterangepicker': $.proxy(this.elementChanged, this),
+                // odoo customization, we don't want this function to execute his normal behaviour
+                // as daterangepicker does not know which input has been modified (start, end) and it
+                // does not have reference to the other input.
+                // 'keyup.daterangepicker': $.proxy(this.elementChanged, this),
                 'keydown.daterangepicker': $.proxy(this.keydown, this) //IE 11 compatibility
```

Comment:
why IE 11 compatibility? Where does that come from?

Reply:
The comment comes from the lib directly https://github.com/dangrossman/daterangepicker/blob/master/daterangepicker.js#L434

It's just to hide the daterange picker when esc, enter, tab are pressed. (maybe supported natively) on other browsers

Reply:
Finally tried to comment this line and it just deactivated the feature for all the browsers.
I removed the comment as it is just confusing